### DISCUSSION TITLE
Support for more (Open Access) Paper sites

### DISCUSSION THREAD

**Discussion by BradKML** [SakanaAI/AI-Scientist#165] [Ideas]
Coming from writing a list from the other projects, it would be great if places like OA.mg, InternetArchive, and Unpaywall gets added into the list of places for papers. https://github.com/SmartManoj/Kevin/discussions/136#discussioncomment-11634779
P.S. what other forms of optimizations are there (e.g. updated OpenReview) that makes generating new papers better? https://github.com/SmartManoj/Kevin/discussions/143 https://github.com/SmartManoj/Kevin/discussions/136#discussioncomment-11613013

Currently it is in https://github.com/SakanaAI/AI-Scientist/blob/main/ai_scientist/generate_ideas.py with `search_for_papers` but since there are no standards API across sites, it might be harder to wrangle this. At the end we are only getting paper title, author(s), venue, year of publication, citation count, and paper abstract. Note: it reads abstract but not necessarily content that enables stronger idea synthesis through deeper content summaries.

A bit of a personal self-note on the progress of things https://github.com/SakanaAI/AI-Scientist/pull/135

P.S. @MarlNox has done it for https://core.ac.uk and the code can be retrofitted for other sites as well https://github.com/SakanaAI/AI-Scientist/issues/104#issuecomment-2330981813

**Comment by BradKML** [comment]
List of issues with problems in SemanticScholar https://github.com/SakanaAI/AI-Scientist/issues/18 https://github.com/SakanaAI/AI-Scientist/issues/78 https://github.com/SakanaAI/AI-Scientist/issues/84 https://github.com/SakanaAI/AI-Scientist/issues/118 https://github.com/SakanaAI/AI-Scientist/issues/122 https://github.com/SakanaAI/AI-Scientist/issues/133

**Comment by erkinalp** [comment]
don't forget sci-hub.ru

**Reply by BradKML** [reply]
Please, we need search engines rather than just paper repositories. Also, very likely SakanaAI can't vouch for anything that is not Open Access compliant. https://github.com/SmartManoj/Kevin/discussions/136#discussioncomment-11626569

**Reply by erkinalp** [reply]
In most of the world, AI-generated works get "creator isn't a legal person" treatment hence disqualified from copyright; with the exception of the UK and few ex-British colonies where they instead get work-of-hire treatment.

**Reply by BradKML** [reply]
I am referring to the acquisition of papers in the same way as the OG founder of Reddit (look it up btw), and how future organizations (especially startups with investors) are not allowed to break compliance and end up going the same direction.
